python/chilldkg_ref/chilldkg.py

```diff
@@ -124,44 +125,34 @@ def certeq_coordinator_step(sigs: List[bytes]) -> bytes:
 ###
 
 
-def hostseckey(seed: bytes) -> bytes:
-    return prf(seed, "chilldkg hostseckey")
+def hostpubkey_gen(hostseckey: bytes) -> bytes:
```

Comment:
```suggestion
def hostpubkey(hostseckey: bytes) -> bytes:
```
I like this a bit more. 
 - I don't think the public key is "generated" at this point. It's rather that the entire key pair is generated when the hostseckey is created.
 - For consistency with the other names.  We also don't use `certeq_message_gen` or `params_id_gen`.

Reply:
done. I renamed the local conflicting variables to `hostpk`.